src/__tests__/circles.js

```diff
@@ -0,0 +1,73 @@
+import {toBeDeepCloseTo} from 'jest-matcher-deep-close-to';
+import {generateDataset2D} from '../generateDataset2D';
+expect.extend({toBeDeepCloseTo});
+
+describe('circles test', () => {
+    var options = {
+        samples: 10,
+        kind: 'circles'
+    };
+
+    test('make circles', () => {
+        options.shuffle = false;
+        var {dataset, labels} = generateDataset2D(options);
+
+        var expectedDataset = [[1, 0],
```

Comment:
this is based on what?

Reply:
![deepinscreenshot_select-area_20170907085925](https://user-images.githubusercontent.com/7885709/30150233-eac94fc0-93aa-11e7-90c6-c7a834c2394d.png)

on this.